w06h01/test/pgdp/LoopTest.java

```diff
@@ -36,9 +43,11 @@ public void testUsageOfLoops() {
             String filteredFile = removeMultipleMethods(file, allowedMethods);
 
             // Check if there are any loops
-            for (String keyword : notAllowedKeywords) {
-                assertTrue(!filteredFile.contains(keyword),
-                        "You are not allowed to use loops in your solution! Found a loop : " + keyword);
+            for (String regExp : notAllowedRegExp) {
+                Pattern pattern = Pattern.compile(regExp);
+                Matcher matcher = pattern.matcher(filteredFile);
+                assertTrue(!matcher.find(),
```

Comment:
assertFalse(.., ..) would be a bit more readable and understandable than asserting for a negated false value.

Reply:
oh yes 👍